src/test/regress/expected/alter_table_null_dist_key.out

```diff
@@ -0,0 +1,75 @@
+CREATE SCHEMA alter_null_dist_key;
+SET search_path TO alter_null_dist_key;
+SET citus.next_shard_id TO 1720000;
+SET citus.shard_count TO 32;
+SET citus.shard_replication_factor TO 1;
+CREATE SEQUENCE dist_seq;
+CREATE TABLE null_dist_table(a bigint DEFAULT nextval('dist_seq') UNIQUE, "b" text, c bigint GENERATED BY DEFAULT AS IDENTITY);
+INSERT INTO null_dist_table("b") VALUES ('test');
+SELECT create_distributed_table('null_dist_table', null, colocate_with=>'none', distribution_type=>null);
+NOTICE:  Copying data from local table...
+NOTICE:  copying the data has completed
+DETAIL:  The local data in the table is no longer visible, but is still on disk.
+HINT:  To remove the local data, run: SELECT truncate_local_data_after_distributing_table($$alter_null_dist_key.null_dist_table$$)
+ create_distributed_table
+---------------------------------------------------------------------
+
+(1 row)
+
+-- add column
+ALTER TABLE null_dist_table ADD COLUMN d bigint DEFAULT 2;
+SELECT * FROM null_dist_table;
+ a |  b   | c | d
+---------------------------------------------------------------------
+ 1 | test | 1 | 2
+(1 row)
+
+-- alter column type
+ALTER TABLE null_dist_table ALTER COLUMN d TYPE text;
+UPDATE null_dist_table SET d = 'this is a text' WHERE d = '2';
+SELECT * FROM null_dist_table;
+ a |  b   | c |       d
+---------------------------------------------------------------------
+ 1 | test | 1 | this is a text
+(1 row)
+
+-- drop seq column
+ALTER TABLE null_dist_table DROP COLUMN a;
+SELECT * FROM null_dist_table;
+  b   | c |       d
+---------------------------------------------------------------------
+ test | 1 | this is a text
+(1 row)
+
+-- add not null constraint
+ALTER TABLE null_dist_table ALTER COLUMN b SET NOT NULL;
+-- not null constraint violation, error out
+INSERT INTO null_dist_table VALUES (NULL, 2, 'test');
+ERROR:  null value in column "b" violates not-null constraint
+DETAIL:  Failing row contains (null, 2, test).
+CONTEXT:  while executing command on localhost:xxxxx
+-- drop not null constraint and try again
+ALTER TABLE null_dist_table ALTER COLUMN b DROP NOT NULL;
+INSERT INTO null_dist_table VALUES (NULL, 3, 'test');
+SELECT * FROM null_dist_table;
+  b   | c |       d
+---------------------------------------------------------------------
+ test | 1 | this is a text
+      | 3 | test
+(2 rows)
+
+-- add exclusion constraint
+ALTER TABLE null_dist_table ADD CONSTRAINT exc_b EXCLUDE USING btree (b with =);
```

Comment:
1. Can we issue rename constraint command right after this?

2. Let's also please add an unnamed version of it:
```sql
ALTER TABLE null_dist_table ADD EXCLUDE USING btree (b with =);
```